specification/apimanagement/resource-manager/Microsoft.ApiManagement/stable/2017-03-01/apimversionsets.json

```diff
@@ -0,0 +1,383 @@
+                        "required": true,
+                        "description": "The entity state (Etag) version of the Api Version Set to delete. A value of \"*\" can be used for If-Match to unconditionally apply the operation.",
+                        "type": "string"
+                    }
+                ],
+                "responses": {
+                    "204": {
+                        "description": "The user details were successfully deleted."
+                    },
+                    "default": {
+                        "description": "Error response describing why the operation failed.",
+                        "schema": {
+                            "$ref": "./apimanagement.json#/definitions/ErrorResponse"
+                        }
+                    }
+                }
+            }
+        }
+    },
+    "definitions": {
+        "ApiVersionSetContractProperties": {
+            "description": "An API Version Set contains the common configuration for a set of API Versions relating ",
+            "properties": {
+                "id": {
+                    "type": "string",
+                    "description": "Identifier for existing API Version Set. Omit this value to create a new Version Set."
+                },
+                "description": {
+                    "type": "string",
+                    "description": "Description of API Version Set."
+                },
+                "versioningScheme": {
+                    "type": "string",
+                    "description": "An value that determines where the API Version identifer will be located in a HTTP request.",
+                    "enum": [
+                        "Segment",
+                        "Query",
+                        "Header"
+                    ]
+                },
+                "versionQueryName": {
+                    "type": "string",
+                    "description": "Name of query parameter that indicates the API Version if versioningScheme is set to `query`."
+                },
+                "versionHeaderName": {
+                    "type": "string",
+                    "description": "Name of HTTP header parameter that indicates the API Version if versioningScheme is set to `header`."
+                }
+            }
+        },
+        "ApiVersionSetContract": {
+            "properties": {
+                "properties": {
+                  "x-ms-client-flatten": true,
+                  "$ref": "#/definitions/ApiVersionSetContractProperties",
+                  "description": "Properties of the Api Version Set."
+                }
+              },
+              "allOf": [
+                {
+                  "$ref": "./apimanagement.json#/definitions/Resource"
+                }
+              ],
+              "description": "Api Version Set Contract details."
+        }
+    },
+    "parameters": {
+        "ApiVersionSetIdParameter": {
+            "name": "uid",
```

Comment:
>uid [](start = 21, length = 3)

versionSetId

replace uid in the file with versionSetId. That way we will be consistent with our smapi controller and the automation I built for generating these specs